task5/src/main/java/com/clouway/adapter/http/BankOperationHandler.java

```diff
@@ -0,0 +1,53 @@
+package com.clouway.adapter.http;
+
+import com.clouway.core.UserRepository;
+import com.clouway.core.Validator;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * Created by Kristiyan Petkov  <kristiqn.l.petkov@gmail.com> on 08.06.16.
+ */
+@WebServlet(name = "BankOperationHandler")
+public class BankOperationHandler extends HttpServlet {
+  private UserRepository userRepository;
+  private Validator validator;
+
+
+  public BankOperationHandler(UserRepository userRepository, Validator validator) {
+    this.userRepository = userRepository;
+    this.validator = validator;
+  }
+
+  protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+    String operation = request.getParameter("operation");
+    String email = request.getParameter("email");
```

Comment:
This looks like security leak. Here we have to use something userrepository.getCurrentUser(sessionId); this is the common practise;